hydra-node/src/Hydra/Ledger/Cardano.hs

```diff
@@ -591,6 +627,33 @@ genOneUtxoFor vk = do
   output <- scale (const 1) $ genOutput vk
   pure $ Utxo $ Map.singleton (fromShelleyTxIn input) output
 
+-- | Generate UTXO entries that do not contain any assets. Useful to test /
+-- measure cases where
```

Comment:
> cases where ....
?

Reply:
Ah! That's the moment I went to answer the door I guess.

Cases where pure ada payments form a valid use-case on their own.